configure

```diff
@@ -340,6 +340,7 @@ pvrdma=""
 gprof="no"
 debug_tcg="no"
 tcg_log_instr="yes"
+tcg_log_instr_perfetto="yes"
```

Comment:
This should probably be off by default as we pull in some of the boost libraries and the perfetto submodule as a dependency.